docs/l4-l7.rst

```diff
@@ -130,14 +134,16 @@ Update an application using Advanced mode (Application subtab)
 
 4. Update the application JSON.
 
-5. Optionally, Click on :guilabel:`Manage Endpoint Mappings` to update Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
+5. Optionally, Click on :guilabel:`Manage Endpoint Mappings` to update Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool.The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
 
 6. Click :guilabel:`Submit`.
 
 The application is updated on the BIG-IP device.
 
 .. note::
 
+   - Route Domain must be created on the BIG-IP since the L2-L3 Network Management Tab as well as AS3 configuration does not support creation of Route Domains.
```

Comment:
... the L2-L3 Network Management tab, as well as the AS3 configuration, does not....